src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/runMatlabTests.m

```diff
@@ -2,22 +2,24 @@
 
 function failed = runMatlabTests(varargin)
 
+
 p = inputParser;
-p.addParameter('PDFReport', false, @islogical);
-p.addParameter('TAPResults', false, @islogical);
-p.addParameter('JUnitResults', false, @islogical);
-p.addParameter('SimulinkTestResults', false, @islogical);
-p.addParameter('CoberturaCodeCoverage', false, @islogical);
-p.addParameter('CoberturaModelCoverage', false, @islogical);
+p.addParameter('PDFReportPath', '', @ischar);
+p.addParameter('TAPResultsPath', '', @ischar);
+p.addParameter('JUnitResultsPath', '', @ischar);
+p.addParameter('SimulinkTestResultsPath', '', @ischar);
+p.addParameter('CoberturaCodeCoveragePath', '', @ischar);
+p.addParameter('CoberturaModelCoveragePath', '', @ischar);
```

Comment:
`@ischar` will return true for char matrices. You need to further constrain it so it can only be a char row vector. Something like `@(c)ischar(c) && (isempty(c) || isrow(c))`.

Reply:
Do you see any possibility of char matrices being passed ? . unless used this this file in isolation without our plugin.

Reply:
I tend to think a well design component should work in isolation. Sometimes some cost/benefit analysis to consider but this one seems fairly low cost.

Reply:
Done.